main.js

```diff
@@ -0,0 +1,34 @@
+const csv = require('csv-parser');
+const fs = require('fs');
+const results = []; //to store the result of the csv file here!

```

Comment:
What `results` mean here, its a very generic name, and meaningless in this context.
We should use meaning full names for our variables, functions.